birdbox/common/templatetags/birdbox_tags.py

```diff
@@ -77,3 +78,8 @@ def gather_field_errors(form):
             errors.append(f"{field.label}: {combined_errors}")
 
     return errors
+
+
+@register.simple_tag
+def should_use_sso_auth():
+    return settings.USE_SSO_AUTH is True
```

Comment:
is this better than just `return settings.USE_SSO_AUTH`?

Reply:
No, just me being a doofus and leaving in unnecessary code. Will trim!